core/services/ocr3/plugins/ccipevm/executecodec.go

```diff
@@ -0,0 +1,179 @@
+package ccipevm
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/evm_2_evm_multi_offramp"
+	"github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/ccip/abihelpers"
+)
+
+// ExecutePluginCodecV1 is a codec for encoding and decoding execute plugin reports.
+// Compatible with:
+// - "EVM2EVMMultiOffRamp 1.6.0-dev"
+type ExecutePluginCodecV1 struct {
+	executeReportEventInputs abi.Arguments
+}
+
+func NewExecutePluginCodecV1() *ExecutePluginCodecV1 {
+	abiParsed, err := abi.JSON(strings.NewReader(evm_2_evm_multi_offramp.EVM2EVMMultiOffRampABI))
+	if err != nil {
+		panic(fmt.Errorf("parse multi offramp abi: %s", err))
+	}
+	eventInputs := abihelpers.MustGetMethodInputs("manuallyExecute", abiParsed)
```

Comment:
nit: `methodInputs`?

```suggestion
	methodInputs := abihelpers.MustGetMethodInputs("manuallyExecute", abiParsed)
```